specification/purview/data-plane/Azure.Analytics.Purview.Scanning/stable/2023-09-01/scanningService.json

```diff
@@ -0,0 +1,9730 @@
+          "$ref": "#/definitions/Scan"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/OracleOracleCredentialScanProperties"
+        }
+      },
+      "x-ms-discriminator-value": "OracleOracleCredential"
+    },
+    "OracleOracleUserPassScanProperties": {
+      "description": "Oracle user pass scan properties.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/MitiScanProperties"
+        }
+      ],
+      "properties": {
+        "username": {
+          "description": "The username of Oracle server.",
+          "type": "string"
+        },
+        "password": {
+          "description": "The password of Oracle server.",
+          "type": "string"
+        },
+        "schema": {
+          "description": "The schema.",
+          "type": "string"
+        },
+        "driverLocation": {
+          "description": "The driver location.",
+          "type": "string"
+        }
+      }
+    },
+    "OracleOracleUserPassScan": {
+      "description": "Oracle user pass scan.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/Scan"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/OracleOracleUserPassScanProperties"
+        }
+      },
+      "x-ms-discriminator-value": "OracleOracleUserPass"
+    },
+    "SapS4HanaSapS4HanaCredentialScanProperties": {
+      "description": "SAP S/4HANA credential scan properties.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/MitiScanProperties"
+        }
+      ],
+      "properties": {
+        "clientId": {
+          "description": "The client ID of SAP S/4HANA server.",
+          "type": "string"
+        },
+        "credential": {
+          "$ref": "#/definitions/CredentialReference"
+        },
+        "jCoLibraryPath": {
+          "description": "The JCo library path.",
+          "type": "string"
+        }
+      }
+    },
+    "SapS4HanaSapS4HanaCredentialScan": {
+      "description": "SAP S/4HANA credential scan.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/Scan"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/SapS4HanaSapS4HanaCredentialScanProperties"
+        }
+      },
+      "x-ms-discriminator-value": "SapS4HanaSapS4HanaCredential"
+    },
+    "SapS4HanaSapS4HanaUserPassScanProperties": {
+      "description": "SAP S/4HANA user pass scan properties.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/MitiScanProperties"
+        }
+      ],
+      "properties": {
+        "clientId": {
+          "type": "string"
+        },
+        "username": {
+          "type": "string"
+        },
+        "password": {
+          "type": "string"
+        },
+        "jCoLibraryPath": {
+          "type": "string"
+        }
```

Comment:
these... Please try do the fixes yourself, instead asking reviewer point every case.

Reply:
Sure, added this description.